### PR TITLE
Fix gtk-update-icon-cache build

### PR DESCRIPTION
**Pull request by muntyan**
I don't know how to use git and/or github, so this pull request contains three things, but I am only requesting to pull the gtk+.sln change (see the commit message and linked ms bug for details).

### CODE REVIEW

**Comment by nacho** [comment]
Hey, i gave u permissions to push upstream sometime ago. Just add the upstream remote and push directly

**Comment by muntyan** [comment]
You really overestimate my git skills :) In this particular case there is only one change that needs to be pushed up, and I don't know how to do that. Two previous changes are garbage, at least for now.

**Comment by nacho** [comment]
On this pull request there are 3 commits. Another way is that u make a new clone of the repo using the upstream URL. Then just push the change to master.

**Comment by nacho** [comment]
If u are not able I will try to cherry pick the commit when I have a computer with me

**Comment by nacho** [comment]
Actually are you sure about this? See: https://git.gnome.org/browse/gtk+/tree/build/win32/vs10/gtk+.sln?h=gtk-2-24#n15 I guess you just need to update gtk to the new version

**Comment by muntyan** [comment]
This change is for the solution file in gtk-win32, maybe it needs to be removed if upstream solution is good?

File: build.py [hunk 1]
```diff
@@ -931,6 +931,48 @@ def post_install(self):
...
+    def __init__(self):
+        Project.__init__(self,
+            'glibmm',
+            archive_url = 'http://ftp.gnome.org/pub/GNOME/sources/glibmm/2.50/glibmm-2.50.0.tar.xz',
+            dependencies = ['glib', 'libsigcpp'],
+            patches = ['props.patch'],
+            )
+
+    def build(self):
+        self.exec_msbuild(r'MSVC_Net2013\glibmm.sln')
```
Comment (nacho) [inline build.py]: you should also install the licenses

**Comment by nacho** [comment]
Untested, but here you have a possible update https://github.com/wingtk/gtk-win32/tree/wip/gtk-2.24.31

**Comment by nacho** [comment]
I just fixed this upstream by updating to the last gtk 2.24.31 release.

**Pull request closed without merging**